clog/readers.py

```diff
@@ -284,6 +287,8 @@ def __init__(self,
                 raise Exception("Last n lines can be only used with new kafka tailer")
             self._stream = stream + " " + str(lines)
             self._automagic_recovery = False
+        if protocol_opts:
+            self._stream += ''.join([' {0}={1}'.format(k, v) for k, v in protocol_opts.items()])
```

Comment:
Maybe it's possible to do this without modifying _stream? I think it might be confusing if we need to read the stream name later.